### PR TITLE
Salwa's pull request for making Cow class

### PR DESCRIPTION
**Pull request by sms-sudo**
Created new Cow class that implements tradeable and domesticatable
Includes test file
Changed trader to initialize a Cow

### CODE REVIEW

File: src/test/java/CowTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/*
+ * This file contains sample JUnit test cases for Horse.java
```
Comment (marianmwang) [inline src/test/java/CowTest.java]: this is for Cow.java

File: src/test/java/CowTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    public void TestGetPrice() {
+        assertEquals(24, c.getPrice());
+    }
+
+    @Test(timeout = 50)
+    public void TesttoString() {
+        assertEquals("I'm a cow and the noise I make is Moo!", c.toString());
+    }
+
+}
```
Comment (marianmwang) [inline src/test/java/CowTest.java:39]: misisng tests! should include test for your new SetColour method

File: src/main/java/Cow.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/* A sample class that implements the given interfaces.
+ */
+public class Cow implements Domesticatable, Tradable {
+    private String colour;
+
+    public Cow() {
+        this.colour = "brown";
+    }
+
+    public void Setcolour(String colour) {
```
Comment (marianmwang) [inline src/main/java/Cow.java]: bad naming convention, should be "setColour"

**Review by marianmwang — CHANGES_REQUESTED** [review]

File: src/test/java/CowTest.java (hunk 1 above)
Comment (sms-sudo) [reply src/test/java/CowTest.java]: I just forgot to change it... but I'll count it as an intended error. :))

File: src/test/java/CowTest.java (hunk 2 above)
Comment (sms-sudo) [reply src/test/java/CowTest.java:39]: Yes !

File: src/main/java/Cow.java (hunk 3 above)
Comment (sms-sudo) [reply src/main/java/Cow.java]: Yes !

Comment (Silk8Worm) [reply src/main/java/Cow.java]: ^

**Review by Silk8Worm — APPROVED** [review]

**Pull request merged**